autosklearn/ensemble_builder.py

```diff
@@ -931,19 +931,20 @@ def score_ensemble_preds(self):
                     y_ens_fn,
                     traceback.format_exc(),
                 )
-                self.read_scores[y_ens_fn]["ens_score"] = -np.inf
+                self.read_losses[y_ens_fn]["ens_loss"] = np.inf
 
         self.logger.debug(
             'Done reading %d new prediction files. Loaded %d predictions in '
             'total.',
             n_read_files,
-            np.sum([pred["loaded"] > 0 for pred in self.read_scores.values()])
+            np.sum([pred["loaded"] > 0 for pred in self.read_losses.values()])
         )
+        print(f"self.read_losses={self.read_losses}")
```

Comment:
Could you please remove this print again?

Reply:
Sorry, removed.